bin/migration-scripts/0.14.20.js

```diff
@@ -0,0 +1,31 @@
+export const description = 'v0.14.20';
+
+export default (uniPath, content) => {
+  if (!content?.blocks) {
+    return content;
+  }
+
+  for (const block of content.blocks) {
+    adjustBonusBenefitsBlock(block);
+  }
+
+  return { ...content };
+};
+
+function adjustBonusBenefitsBlock(block) {
+  if (!block?.content) {
+    return;
+  }
+
+  if (block?.type === 'CarouselRecommendationCard' && block.content.cardTextAlign) {
+    if (block.content.cardTextAlign === 'left') {
+      block.content.cardTextAlign = 'text-left';
+    }
+    if (block.content.cardTextAlign === 'center') {
+      block.content.cardTextAlign = 'text-center';
+    }
+    if (block.content.cardTextAlign === 'right') {
+      block.content.cardTextAlign = 'text-right';
+    }
```

Comment:
Такое лучше в мапу вынести. 
и написать что то типа 
```suggestion
    const {cardTextAlign} = block.content;
    block.content.cardTextAlign = MAP[cardTextAlign] || cardTextAlign;
```